src/exchangeRates.js

```diff
@@ -60,7 +60,7 @@ async function get(params) {
 
 async function getMultiple() {
   let allExchangeRates = await db.query(
-    `SELECT from_currency, to_currency, rate, created_at FROM exchange_rates`,
+    `SELECT from_currency, to_currency, rate, created_at FROM exchange_rates LIMIT 10`,
```

Comment:
this is not going to paginate, add page in the URL and an offset here too. then add tests for the same, please!